src/main/resources/application.yml

```diff
@@ -42,8 +42,8 @@ image-manager:
   disk-log-mount-point: ${LOG_DIR}
   log-requests: ${LOG_REQUESTS}
   url-show-mode: true
-  route-to-limit: http://localhost:5000
-  connection-limit: 5
+  route-to-limit: ${ORIGIN_ROUTE}
```

Comment:
can you make it a `routes-to-limit` and then all the origins could be passed in a format of 
`http://a.com,http://b.com,http://origin` (separated by a comma)  and so on? And adjust the connection manager accordingly so it takes several routes into consideration?